### PR TITLE
Jbaker/open source compliance

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #6

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 name: Feature request
 description: Suggest an idea for this project
-title: 'auro-formvalidation: [feature summary]'
+title: 'wc-generator: [feature summary]'
```
Comment (blackfalcon) [inline .github/ISSUE_TEMPLATE/feature_request.yml:3]: revert

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 labels:
   - 'Type: Feature'
   - not-reviewed
-  - 'auro-formvalidation'
+  - wc-generator
```
Comment (blackfalcon) [inline .github/ISSUE_TEMPLATE/feature_request.yml:7]: revert

File: .github/ISSUE_TEMPLATE/general-support.yml [hunk 3]
```diff
@@ -1,11 +1,11 @@
 name: General support
 description: Suggestions that are not related to bugs or new features
-title: 'auro-formvalidation: [issue]'
+title: 'wc-generator: [issue]'
```
Comment (blackfalcon) [inline .github/ISSUE_TEMPLATE/general-support.yml:3]: revert

File: .github/ISSUE_TEMPLATE/general-support.yml [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 labels:
   - 'Type: Question'
   - 'Type: Feature'
   - not-reviewed
-  - 'auro-formvalidation'
+  - wc-generator
```
Comment (blackfalcon) [inline .github/ISSUE_TEMPLATE/general-support.yml:8]: revert

File: scripts/generateDocs.cjs [hunk 5]
```diff
@@ -4,6 +4,11 @@ const path = require('path');
 const markdownMagic = require('markdown-magic');
 const fs = require('fs');
 
+// import path from 'path';
+// import markdownMagic from 'markdown-magic';
+// import fs from 'fs';
+// import https from 'https';
+
```
Comment (blackfalcon) [inline scripts/generateDocs.cjs:11]: Please delete the commented code.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: web-test-runner.config.mjs [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+  nodeResolve: true,
+  coverageConfig: {
+    threshold: {
+      statements: 0,
+      branches: 0,
+      functions: 0,
+      lines: 0
+    }
+  }
+};
```
[the remark this answers is not in the document]
Comment (jason-capsule42) [reply web-test-runner.config.mjs:12]: All validation is tested within the components that use the library. This repo doesn't contain the necessary code to establish stored values on a parent component, nor any attributes necessary to define the validation paramaters (required, maxlength, min, pattern, etc).

That's why all testing is within the parent components and there is no scenario where this code is "in production" without being used and validated by the parent component.

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 2 above)
Comment (jason-capsule42) [reply .github/ISSUE_TEMPLATE/feature_request.yml:7]: Why would this repo title the tickets as though they were part of the wc-generator? This is not a pattern we follow anywhere else other than the actual generator.

File: .github/ISSUE_TEMPLATE/general-support.yml (hunk 3 above)
Comment (jason-capsule42) [reply .github/ISSUE_TEMPLATE/general-support.yml:3]: Why would this repo title the tickets as though they were part of the wc-generator? This is not a pattern we follow anywhere else other than the actual generator.

File: .github/ISSUE_TEMPLATE/general-support.yml (hunk 4 above)
Comment (jason-capsule42) [reply .github/ISSUE_TEMPLATE/general-support.yml:8]: Why would this repo title the tickets as though they were part of the wc-generator? This is not a pattern we follow anywhere else other than the actual generator.

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 1 above)
Comment (blackfalcon) [reply .github/ISSUE_TEMPLATE/feature_request.yml:3]: This is your commit from March 8th https://github.com/AlaskaAirlines/auro-formvalidation/commit/fd17eb91165c1c79d8944b813b40601495fd2945

You changed `auro-formvalidation` to `wc-generator`. I assume that you copied this template from the generator and forgot to change this back?

File: web-test-runner.config.mjs (hunk 6 above)
[the remark this answers is not in the document]
Comment (blackfalcon) [reply web-test-runner.config.mjs:12]: Hmmmm, that is dangerous. Perhaps there is a way to create mock code for testing purposes.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

**Review by jordanjones243 — APPROVED** [review]

**Pull request merged**